public/app/features/integrated-alerting/components/NotificationChannel/AddNotificationChannelModal/PagerDutyFields/PagerDutyFields.tsx

```diff
@@ -1,18 +1,31 @@
 import React, { FC } from 'react';
-import { TextInputField, validators } from '@percona/platform-core';
+import { TextInputField } from '@percona/platform-core';
 import { Messages } from '../AddNotificationChannelModal.messages';
 import { PagerDutyFieldsProps } from './PagerDutyFields.types';
 
-export const PagerDutyFields: FC<PagerDutyFieldsProps> = ({ values }) => {
-  const { required } = validators;
-  const { routing, service } = values;
-  const routingValidators = service ? [] : [required];
-  const serviceValidators = routing ? [] : [required];
+/**
+ * This is a validator factory.
+ * Given a key, return a validator that will check if there's anything there.
+ */
+const anotherKeyRequired = (complementaryKey: string) => (value: any, allValues: any): undefined | string => {
+  if (allValues[complementaryKey]) {
+    /**
+     * The complementary field is there, we're good to go without errors
+     */
```

Comment:
```suggestion
     // The complementary field is there, we're good to go without errors
```
The multi-line comments work best when there multiple lines, so let's save some space :)

Reply:
👍